.github/workflows/static-analysis.yml

```diff
@@ -23,10 +23,6 @@ jobs:
           - "default"
         persistence-version:
```

Comment:
Should we remove this as well? I don't think we will test for Persistence 4 very soon.

Reply:
Good idea. Even if we do, we can still copy the config from the 2.5.x branch.